assets/vhs/Dockerfile

```
@@ -0,0 +1,82 @@
+# syntax=docker/dockerfile:1.7
+
+FROM rust:bookworm
+
+COPY rust-toolchain.toml /tmp/rust-toolchain.toml
+
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)